kids_books.py

```diff
@@ -0,0 +1,174 @@
+# Task: create an application that will store the content of our library. I chose children's books.
+# more info about task is in the file "zadani_ukolu.txt"
+
+import click
+from datetime import datetime
+from sqlalchemy import create_engine
+from sqlalchemy.orm import declarative_base
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import Column, Integer, String, DateTime, Boolean
+
+db = create_engine("sqlite:///books.sqlite")
+Base = declarative_base()
+
+## Create empty database
+def connect_to():
+    Base.metadata.create_all(db)
+    Session = sessionmaker(bind=db)
+    return Session()
+
+class Book(Base):
+    __tablename__ = "kids_books"
+    id = Column(Integer, primary_key=True)
+    book_name = Column(String)
+    author = Column(String)
+    number_of_pages = Column(Integer)
+    date_added = Column(DateTime)
+    read_yes_no = Column(Boolean, default=False)
+    partially_read = Column(Boolean, default=False)
+    lent_to = Column(String)        # name of person or None
+    date_of_borrowing = Column(DateTime)
+    notes = Column(String)
+
+    def __repr__(self):
+        return f"<Kniha(id={self.id}, book_name='{self.book_name}', author = '{self.author}', date_added={self.date_added})>"
+
+
+@click.group()
+def kids_library():
+    pass
+
+
+## ADD book into the library
+@kids_library.command()
+@click.option("--name", prompt="Name of the book")
+@click.option("--author", prompt="Author of the book")
+@click.option("--pages", prompt="Number of pages")
+def add_book(name, author, pages):
+    book_session = connect_to()
+    book = Book(book_name=name, author=author, number_of_pages=pages, date_added=datetime.now())
+    book_session.add(book)  
+    book_session.commit()
+    print(f"Added: {book.book_name}, {book.author}, {book.number_of_pages}")
+
+
+## SUMMARY, write out the content of the library, dump file
+# vylepšit výpis půjčených knih, aby se vypsalo komu jsou půjčené
+# přidat možnost výpisu poznámek
+@kids_library.command()
+@click.option("--borrowed", default=False, is_flag=True)   # co vlastne dělá is_flag a z jakeho duvodu se tam dava
+def dump_file(borrowed):
+    book_session = connect_to()
+    qu = book_session.query(Book)   #qu (=query)
+    if borrowed:
+        qu = qu.filter(Book.lent_to != None)
+        # print(f"{Book.id} {Book.book_name} PŮJČENO uživateli {Book.lent_to}")     ## nevypíší se konkrétní hodnoty ale ty zavorky
```

Comment:
Nevypisi, protoze si jeste musis nejprve vsechny odpovidajici radky vyvolat tak, jak to mas na r.70